examples/mnist_00.md

```diff
@@ -0,0 +1,242 @@
+```python
+cs = ConfigStore.instance()
+cs.store(name="config", node=MNISTConf)
+```
+The name `config` will be passed to the `@hydra` decordator when we get to `main()`.
+
+***
+### Parting with Argparse
+
+Now we're starting to realize our relationship with `argparse` isn't as serious as we thought it was. Although `argparse` is powerful, we can take it a step further. In the process we hope to introduce greater organization and free our primary file from as much boilerplate as possible.
+
+One feature Hydra provides us is aggregating our configuration files alongside any 'specifications' we pass via command line arguments. What this means is as long as we have the configuration file which defines possible arguments like `save_model` or `dry_run`, there is no need to also litter our code with `argparse` definitions.
+
+This whole block in `main()`:
+```python
+def main():
+# Training settings
+    parser = argparse.ArgumentParser(description='PyTorch MNIST Example')
+    parser.add_argument('--batch-size', type=int, default=64, metavar='N',
+                        help='input batch size for training (default: 64)')
+    parser.add_argument('--test-batch-size', type=int, default=1000, metavar='N',
+                        help='input batch size for testing (default: 1000)')
+    parser.add_argument('--epochs', type=int, default=14, metavar='N',
+                        help='number of epochs to train (default: 14)')
+    parser.add_argument('--lr', type=float, default=1.0, metavar='LR',
+                        help='learning rate (default: 1.0)')
+    parser.add_argument('--gamma', type=float, default=0.7, metavar='M',
+                        help='Learning rate step gamma (default: 0.7)')
+    parser.add_argument('--no-cuda', action='store_true', default=False,
+                        help='disables CUDA training')
+    parser.add_argument('--dry-run', action='store_true', default=False,
+                        help='quickly check a single pass')
+    parser.add_argument('--seed', type=int, default=1, metavar='S',
+                        help='random seed (default: 1)')
+    parser.add_argument('--log-interval', type=int, default=10, metavar='N',
+                        help='how many batches to wait before logging training status')
+    parser.add_argument('--save-model', action='store_true', default=False,
+                        help='For Saving the current Model')
+    args = parser.parse_args()
+```
+becomes:
+```python
+def main(cfg):
+# All argparse args now reside in cfg
+```
+Our initial strategy is to dump these arguments directly in our top-level configuration.
+```python
+@dataclass
+class MNISTConf:
+    batch_size: int = 64
+    test_batch_size: int = 1000
+    epochs: int = 14
+    lr: float = 1.0 # REMOVE THIS SINCE IT IS NOW WITHIN `AdadeltaConf` BELOW
+    gamma: float = 0.7 # REMOVE THIS SINCE IT IS NOW WITHIN `AdadeltaConf` BELOW

```

Comment:
I am not sure I like having those extra configs here in the first example with the remove comment.
Instead you can just call out that lr and gamma are a part of the standard AdaDeltaConf object?